task7(28)/task7(28)/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task7_28_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите X");
+            x = double.Parse(Console.ReadLine());
+            Console.WriteLine(f(x));
+            Console.ReadKey();
+        }
+        static double f (double x)
```

Comment:
Методы принято называть с большой буквы
```suggestion
        static double F(double x)
```